community/values/src/main/java/org/neo4j/values/utils/InvalidValuesArgumentException.java

```diff
@@ -19,7 +19,12 @@
  */
 package org.neo4j.values.utils;
 
-
+/**
+ * {@code InvalidValuesArgumentException} is thrown when trying to pass in an invalid argument to
```

Comment:
I like! Using "@link" instead of "@code" would give you the benefit of being able to jump to the non-existing JavaDocs of the classes you're referencing here, but since it is non-existing, that can as well be skipped, I guess.